ex/parser/nodes.py

```diff
@@ -8,6 +9,7 @@
 from Vintageous.ex.parser.tokens import TokenPercent
 from Vintageous.ex.parser.tokens import TokenSearchBackward
 from Vintageous.ex.parser.tokens import TokenSearchForward
+from Vintageous.state import State
```

Comment:
Ok, I'm not quite sure about this as I have no background for the definition of 'State' class. It seems the marks are globally accessible but has to go thru a State object that associated with current view.

You are right that we might need to think about it deeper. 

From testing point of view, I'v been using this for a period of time and I'm using marks quite a lot in my daily editing activities, till now didn't see a problem. :-)